Makefile

```diff
@@ -0,0 +1,4 @@
+markdownlint: mdl
+
+mdl:
+	mdl --style .markdownlint/style.rb deploy/docs
```

Comment:
same ask like for `yamllint`, please add it to vagrant's Makefile, so we can use `sumo-make`